modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -159,7 +159,10 @@ function BulkActions({
 				selectable,
 				sidePanelId,
 			}) => (
-				<nav className="management-bar management-bar-primary navbar navbar-expand-md">
+				<nav
+					className="management-bar management-bar-primary navbar navbar-expand-md"
+					data-qa-id="selectionToolbar"
+				>
```

Comment:
We should migrate this to `{ManagementToolbar} from 'frontend-js-components-web'` at some point :disappointed: